.github/workflows/ci.yml

```diff
@@ -0,0 +1,89 @@
+name: CI
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build:
+    name: "Build"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04]
+        include:
+          - os: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build sourcemod plugin
+        uses: maxime1907/action-sourceknight@v1
+        with:
+          cmd: build
+
+      - name: Create package
+        run: |
+          mkdir -p /tmp/package
+          cp -R .sourceknight/package/* /tmp/package
+          cp -R addons/sourcemod/gamedata /tmp/package/common/addons/sourcemod/
```

Comment:
The "common" directory shouldn't be making its way into artifact/release packages.

Reply:
the common folder is the default one for when compiling with our docker-sourceknight:
- https://github.com/srcdslab/docker-sourceknight/blob/f27b0b27e2213f8fcf0ad289b2e0c15d712cb7c3/python/3.10-slim/entrypoint.sh#L24

This separation has been made for plugins supporting multiple games but requiring different resources (models, materials, sounds) like here:
- https://github.com/srcdslab/sm-plugin-TopDefenders